VideoApp/VideoApp/TwilioVideo/LocalParticipant/LocalParticipantManager.swift

```diff
@@ -102,15 +106,15 @@ extension LocalParticipantManager: LocalParticipantDelegate {
         videoTrack: LocalVideoTrack,
         error: Error
     ) {
-        errorPublisher.send(error)
+        print("Failed to publish video track: \(error)")
```

Comment:
Just logging these errors instead of publishing them. It can get tricky handling non-fatal errors like this and it seems like log is sufficient.